script/translations/download.py

```diff
@@ -76,9 +82,12 @@ def filter_translations(translations: dict[str, Any], strings: dict[str, Any]) -
                 continue
 
 
-def save_language_translations(lang, translations):
+def save_language_translations(lang: str, translations: dict[str, Any]) -> None:
     """Save translations for a single language."""
     components = translations.get("component", {})
+
+    flattened_all_translations = flatten_translations(translations)
```

Comment:
I think it's confusing to say `flattened_all_translations` if there's not another variable that is called `all_translations`.
```suggestion
    flattened_translations = flatten_translations(translations)
```